src/main/java/org/mifos/connector/gsma/account/AccountResponseProcessor.java

```diff
@@ -47,7 +47,7 @@ public void process(Exchange exchange) {
 
         zeebeClient.newPublishMessageCommand()
                 .messageName(ACCOUNT_STATUS)
-                .correlationKey(exchange.getProperty(CORELATION_ID, String.class))
+                .correlationKey(exchange.getProperty(CORRELATION_ID, String.class))
                 .timeToLive(Duration.ofMillis(timeToLive))
```

Comment:
Hope you are using the support API for responses in all the async flows to recover from network failures during callbacks.

Reply:
Yes, we use transaction state worker during retries if the transaction call goes through.